include/merlin/core_kernels/upsert.cuh

```diff
@@ -1146,6 +1146,7 @@ struct KernelSelector_Upsert {
       }
     };
 
+#if defined(CUDART_VERSION) && (CUDART_VERSION >= 11030)
```

Comment:
Hi @jiashuy, I cleaned some TLPv2 warnings we discussed before. Please help review and confirm. Thanks!